contracts/BeiKeBox.sol

```diff
@@ -24,10 +25,11 @@ contract BeiKeBox is ERC1155 {
         _;
     }
 
-    function mint(address producer, uint id, uint amount) public onlyOwner() {
-        _mint(producer, id, amount, "");
-        ownerMapping[id] = producer;
+    function mint(address producer, uint amount) public onlyOwner() {
```

Comment:
Could you return the token id for me?

Reply:
Sure.

Reply:
Since we call the function in off-chain mode on testing, it should pass the return value from event.

Therefore, I implement the event, and let `mint()` function not only emit the event but also return the value.

See 567c11b and 62fff2e also.

Reply:
Ok! I see.